src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/ConversationController.kt

```diff
@@ -29,6 +29,7 @@ import java.util.UUID
 class ConversationController(
   private val conversationService: ConversationService,
   private val conversationMapper: ConversationsResourceMapper,
+  private val newConversationMapper: ConversationsResourceMapper,
```

Comment:
Not sure I follow this ... does `ConversationResourceMapper` need injecting twice? 🤔  Its the same mapper class isnt it?
In fact, does this controller need changing at all? 🤔

Reply:
No that a mistake - I wrote the newMapper while I was writing it and then deleted the old one. Intellij has refactored and I've forgotten to remove the old one